schema/me/conversation.js

```diff
@@ -59,6 +59,48 @@ export const MessageType = new GraphQLObjectType({
   },
 })
 
+export const AttachmentType = new GraphQLObjectType({
+  name: "AttachmentType",
+  desciption: "Fields of an attachment (currently from Radiation)",
+  fields: {
+    id: {
+      description: "Attachment id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    content_type: {
+      description: "Content type of file.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    file_name: {
+      description: "File name.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    download_url: {
+      descrpition: "URL of attachment.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+})
+
+export const MessageDetailType = new GraphQLObjectType({
+  name: "MessageDetailType",
+  description: "Complete details of a message (currently from Radiation)",
+  fields: {
+    id: {
+      description: "Radiation id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    attachments: {
+      description: "Attachments.",
+      type: new GraphQLList(AttachmentType),
+    },
+    raw_text: {
```

Comment:
like `created_at` timestamps? it should be part of `message` itself, we can add those.

Metadata basically has some information about the message itself, basically tells us if this message was an invoice message or an initial inquiry or etc..